descartes_trajectory/test/cart_trajectory_pt.cpp

```diff
@@ -77,7 +77,7 @@ TEST(CartTrajPt, getPoses)
   std::vector<std::vector<double> >joint_solutions;
 
   ROS_INFO_STREAM("Testing fuzzy pos point");
-  CartesianRobot robot(POS_TOL+2*EPSILON, ORIENT_TOL+2*EPSILON);
+  CartesianRobot robot(POS_TOL+2*EPSILON, 2*M_PI+2*EPSILON);
```

Comment:
Why not set the value of ORIENT_TOL to what it needs to be (2*M_PI apparently) instead of ignoring it?